ui/ducks/send/send.js

```diff
@@ -1907,6 +1917,7 @@ export function updateRecipientUserInput(userInput) {
           useTokenDetection,
           tokenAddressList,
           isProbablyAnAssetContract,
+          smartContractAddress,
```

Comment:
is this being used? Sorry If I'm missing it? `validateRecipientUserInput` doesn't appear to make use of it?

Reply:
no, will remove

Reply:
removed